### PR TITLE
capture (and address) high level comments sent to authors 

### PR DESCRIPTION
**Pull request by louberger**
Also pick up simplified makefile from ip draft

### CODE REVIEW

**Comment by louberger** [comment]
balazs.a.varga@ericsson.com @jounikor  @g3ysx can you take a look...

File: draft-ietf-detnet-dp-sol-mpls-00.xml [hunk 1]
```diff
@@ -181,7 +192,7 @@
 
     <t hangText="PROF">
      A Packet Reorder Function (PROF) re-orders packets within a DetNet flow  that are received out of order.
-     This function may be implemented at an edge or a relay node.
+     This function may be implemented at an edge node or end system.
```
Comment (jounikor) [inline draft-ietf-detnet-dp-sol-mpls-00.xml]: Is there a  specific reason why the relay node cannot do this function anymore?

**Review by jounikor — COMMENTED** [review]
I got one specific comment on reordering and relay functions... comment embedded.

**Comment by louberger** [comment]
My understanding of the architecture definitions are that DetNet end 
systems and edge nodes can provide DetNet service layer functions while 
relay on provide DetNet transport layer functions -- and this is the 
core difference between edge and relay nodes. Am I wrong?

File: draft-ietf-detnet-dp-sol-mpls-00.xml (hunk 1 above)
Comment (louberger) [reply draft-ietf-detnet-dp-sol-mpls-00.xml]: My understanding of the architecture definitions are that DetNet end
systems and edge nodes can provide DetNet service layer functions while
relay on provide DetNet transport layer functions -- and this is the
core difference between edge and relay nodes. Am I wrong?

**Pull request merged**